workspaces/orchestrator/app-config.yaml

```diff
@@ -0,0 +1,95 @@
+app:
+  title: Janus IDP Plugins
```

Comment:
This should probably be changed to something like RHDH Plugins

Reply:
Thanks! Fixed